src/ResponseBuilder.php

```diff
@@ -6,13 +6,24 @@
 
 class ResponseBuilder
 {
+    private $dataObjectBuilder;
+
+    public function __construct(DataObjectBuilder $dataObjectBuilder)
```

Comment:
Same here, can it be backward compatible?

Reply:
Same reason, this is just used internally in this library.

Reply:
I think it's way less internal than one might think. If you want to re-use the same instance of guzzle client, this has been done manually and an upgrade would break it

Reply:
Done in https://github.com/softonic/graphql-client/pull/30